src/test/kotlin/io/thecontext/ci/Data.kt

```diff
@@ -34,6 +34,7 @@ val testPodcast = Podcast(
 )
 
 val testEpisode = Episode(
+        guid = "TestEpisode",
```

Comment:
`Episode ID`

`Test *` just adds clutter to error messages because everything is being prepended with `Test`.

Reply:
👍